macros.go

```diff
@@ -17,251 +12,23 @@ var (
 
 // MacroFunc defines a signature for applying a query macro
 // Query macro implementations are defined by users / consumers of this package
-type MacroFunc func(*Query, []string) (string, error)
+// This is deprecated - new code should use sqlutil.MacroFunc directly
+// Deprecated
```

Comment:
nit: this could just be
```suggestion
// Deprecated: new code should use sqlutil.MacroFunc directly
```
(with all of the other deprecations formatted the same)